lib/created_message_queue_handler.ts

```diff
@@ -102,7 +106,8 @@ export async function handleMessage(
   notificationModel: NotificationModel,
   retrievedMessage: IRetrievedMessageWithoutContent,
   messageContent: IMessageContent,
-  defaultAddresses: Option<NewMessageDefaultAddresses>
+  defaultAddresses: Option<NewMessageDefaultAddresses>,
+  blobService: BlobService
```

Comment:
it's good practice to keep optional parameters at the end of param list, also as a convention I usually first keep the "services" first, so I would place `blobService` right after or before `notificationModel`.

Reply:
Ok